client/locales/i18n.js

```diff
@@ -0,0 +1,8 @@
+import en from './en.json'
+import pt from './pt.json'
```

Comment:
@vincentrohde could you use `pt_BR` instead of `pt`

Because

pt = Portuguese from Portugal
pt_BR = Portuguese from Brazil

:)